core/modules/catalog/store/category/actions.ts

```diff
@@ -268,6 +268,10 @@ const actions: ActionTree<CategoryState, RootState> = {
                 }
               }
             }
+            context.dispatch('addAvailableFilter', {
```

Comment:
Shouldn’t be a mutation?

Reply:
in our case I'm strongly against plain mutations exposed for users. General rule is to use action instead of mutation if you think that any additional logic may be added in the future It's better to have future-proof action rather than changing the API again ;)